### DISCUSSION TITLE
Create mesh other than cylinder, cube, ...

### DISCUSSION THREAD

**Discussion by llfab** [StereoKit/StereoKit#312] [Q&A]
`Mesh` has a couple of functions to create shapes such as cylinder etc. For something like say Cone I believe I would need to create that shape with `SetVerts`, right?

**Comment by maluoi** [comment]
That's correct! `SetVerts` and `SetInds` can be used to create whatever mesh you feel like coding up. The doc page for `SetVerts` even has an example of assembling a heightmap style mesh via code as well :)

https://stereokit.net/Pages/Reference/Mesh/SetVerts.html
